lua/configs/dashboard.lua

```diff
@@ -45,7 +47,7 @@ function M.config()
 
   g.dashboard_custom_footer = {
     " ",
-    " AstroVim loaded " .. plugins_count .. " plugins ",
+    " AstroVim loaded " .. plugin_count .. " plugin ",
```

Comment:
@Aryevang can you make it `plugins` instead `plugin in the end? since there are multiple plugins it would be better if we use plural form :)

Reply:
@kabinspace, sorry about that. That was a wild replace :)